### PR TITLE
Add rkamlah to the participants

### PR DESCRIPTION
**Pull request by rkamlah**

### CODE REVIEW

**Review by maxnoe — CHANGES_REQUESTED** [review]
There are conflicts now, please resolve those either using the github interface (easy) or locally on your computer.

For this, you need to follow the instructions in the lecture for setting up the `upstream` remote and merge the upstream main branch into your branch and resolve the resulting conflicts:

```
# if not yet done
$ git remote add upstream https://github.com/maxnoe/escape2021-git-demo
$ git fetch upstream
$ git merge upstream/main
```

**Comment by rkamlah** [comment]
I tried, However get a warning when running the fetch command. Please see 
the attached lines.

```
$ git remote add upstream https://github.com/maxnoe/escape2021-git-demo
error: remote upstream already exists.
der_r@Mobiles-Einsatzzentrum MINGW64 ~/escape2021-git-demo (add_robert)
$ git fetch upstream
ssh: connect to host git.github.com port 22: Connection timed out
fatal: Could not read from remote repository.
Please make sure you have the correct access rights
and the repository exists.
```

I'm not sure what's the problem here.

**Comment by maxnoe** [comment]
@rkamlah THe first error just says that you already added the remote.

The second error is a connection timeout, so maybe just try it again...

**Comment by rkamlah** [comment]
I added the first one to show, that this should be fine. And the time out is 
what occured when trying to run the command so far. No improvements when 
repeating.

On Wed, 09 Jun 2021 03:42:09 -0700
  Maximilian Nöthe ***@***.***> wrote:
> @rkamlah THe first error just says that you already added the 
>remote.
> 
> The second error is a connection timeout, so maybe just try it 
>again...
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/maxnoe/escape2021-git-demo/pull/33#issuecomment-857588222

Robert Kamlah
Graduate Student, PhD
Physics of the Sun, Optical Solar Physics
Cosmic Magnetic Fields

Telefon: +49 331 7499-406
e-mail: ***@***.***
https://www.aip.de
--------------------------------------------------------------------------------------
Leibniz-Institut für Astrophysik Potsdam (AIP)
An der Sternwarte 16, 14482 Potsdam, Germany

Vorstand: Prof. Dr. Matthias Steinmetz, Wolfram Rosenbach
Stiftung bürgerlichen Rechts
Stiftungsverzeichnis Brandenburg: 26 742-00/7026
---------------------------------------------------------------------------------------

**Comment by maxnoe** [comment]
What does `git remote -v` show? I think you have a typo in the remote address

**Comment by rkamlah** [comment]
$ git remote -v
origin  https://github.com/rkamlah/escape2021-git-demo.git (fetch)
origin  https://github.com/rkamlah/escape2021-git-demo.git (push)
upstream  git.github.com:maxnoe/escape2021-git-demo (fetch)
upstream  git.github.com:maxnoe/escape2021-git-demo (push)

**Comment by maxnoe** [comment]
@rkamlah Yes, that's wrong.

This looks like you tried to add the ssh URL but replaced the `@` with a dot.

IF you want to ssh, you can fix with:

```
git remote set-url upstream git@github.com:maxnoe/escape2021-git-demo 
```

remember that this needs the public key setup in your profile. If you have not done this yet, you can also use https instead:

```
git remote set-url upstream https://github.com/maxnoe/escape2021-git-demo.git 
```

**Comment by rkamlah** [comment]
$ git remote set-url upstream ***@***.***:maxnoe/escape2021-git-demo

der_r@Mobiles-Einsatzzentrum MINGW64 ~/escape2021-git-demo (add_robert)
$ git fetch upstream
***@***.***: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

der_r@Mobiles-Einsatzzentrum MINGW64 ~/escape2021-git-demo (add_robert)
$ git remote -v
origin  https://github.com/rkamlah/escape2021-git-demo.git (fetch)
origin  https://github.com/rkamlah/escape2021-git-demo.git (push)
upstream  ***@***.***:maxnoe/escape2021-git-demo (fetch)
upstream  ***@***.***:maxnoe/escape2021-git-demo (push)

**Comment by maxnoe** [comment]
@rkamlah Yes, this means you didn't setup git / github for ssh. Please set this up or use the https url:

git remote set-url upstream https://github.com/maxnoe/escape2021-git-demo.git

**Comment by rkamlah** [comment]
Hm, not good. I thought, I did. Anyhow. Seems to have worked now. Thank you 
very much for the support
```
git remote set-url upstream 
https://github.com/maxnoe/escape2021-git-demo.git
der_r@Mobiles-Einsatzzentrum MINGW64 ~/escape2021-git-demo (add_robert)
$ git remote -v
origin  https://github.com/rkamlah/escape2021-git-demo.git (fetch)
origin  https://github.com/rkamlah/escape2021-git-demo.git (push)
upstream  https://github.com/maxnoe/escape2021-git-demo.git (fetch)
upstream  https://github.com/maxnoe/escape2021-git-demo.git (push)
der_r@Mobiles-Einsatzzentrum MINGW64 ~/escape2021-git-demo (add_robert)
$ git fetch upstream
remote: Enumerating objects: 235, done.
remote: Counting objects: 100% (88/88), done.
remote: Compressing objects: 100% (40/40), done.
remote: Total 235 (delta 65), reused 50 (delta 48), pack-reused 147
Receiving objects: 100% (235/235), 47.45 KiB | 177.00 KiB/s, done.
Resolving deltas: 100% (135/135), completed with 4 local objects.
 From https://github.com/maxnoe/escape2021-git-demo
  * [new branch]  main  -> upstream/main
der_r@Mobiles-Einsatzzentrum MINGW64 ~/escape2021-git-demo (add_robert)
$ git merge upstream/main
Auto-merging participants.md
CONFLICT (content): Merge conflict in participants.md
Automatic merge failed; fix conflicts and then commit the result.

```

**Comment by maxnoe** [comment]
Ok, now you just need to resolve the conflicts by editing the file, `git add`, `git commit` and `git push` again to update the pull request here

**Comment by rkamlah** [comment]
$ git push
Everything up-to-date



On Wed, 09 Jun 2021 04:10:12 -0700
  Maximilian Nöthe ***@***.***> wrote:
> Ok, now you just need to `git push` again to update the pull request 
>here
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/maxnoe/escape2021-git-demo/pull/33#issuecomment-857605133

Robert Kamlah
Graduate Student, PhD
Physics of the Sun, Optical Solar Physics
Cosmic Magnetic Fields

Telefon: +49 331 7499-406
e-mail: ***@***.***
https://www.aip.de
--------------------------------------------------------------------------------------
Leibniz-Institut für Astrophysik Potsdam (AIP)
An der Sternwarte 16, 14482 Potsdam, Germany

Vorstand: Prof. Dr. Matthias Steinmetz, Wolfram Rosenbach
Stiftung bürgerlichen Rechts
Stiftungsverzeichnis Brandenburg: 26 742-00/7026
---------------------------------------------------------------------------------------

**Comment by maxnoe** [comment]
@rkamlah Did you already resolve the conflicts and create the commit?

**Comment by rkamlah** [comment]
I hope I didn't miss something. It was about entering my name to the list 
make a commit and pull the request, right?
Thats what I did

On Wed, 09 Jun 2021 04:13:14 -0700
  Maximilian Nöthe ***@***.***> wrote:
> @rkamlah Did you already resolve the conflicts and create the 
>commit?
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/maxnoe/escape2021-git-demo/pull/33#issuecomment-857606915

Robert Kamlah
Graduate Student, PhD
Physics of the Sun, Optical Solar Physics
Cosmic Magnetic Fields

Telefon: +49 331 7499-406
e-mail: ***@***.***
https://www.aip.de
--------------------------------------------------------------------------------------
Leibniz-Institut für Astrophysik Potsdam (AIP)
An der Sternwarte 16, 14482 Potsdam, Germany

Vorstand: Prof. Dr. Matthias Steinmetz, Wolfram Rosenbach
Stiftung bürgerlichen Rechts
Stiftungsverzeichnis Brandenburg: 26 742-00/7026
---------------------------------------------------------------------------------------

File: participants.md [hunk 1]
```diff
@@ -28,8 +28,7 @@ Please add your name in alphabetic order
 * @matiaspcerioni
 * @mfschneider
 * @nbiederbeck
-* @oscarcavichia
```
Comment (maxnoe) [inline participants.md:31]: Please don't remove other people ;)

**Review by maxnoe — APPROVED** [review]

**Pull request merged**